accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,790 @@
+   by construction.
+
+2. **Versioning issues**. By making the OS bindings available on top of the .NET
+   platform the consumer is now able to upgrade the .NET platform and the OS
+   bindings independently, which makes it hard to explain what combinations are
+   supported. In practice, the OS bindings will want to depend on .NET types
+   which might change over time (think `Task<T>` or `Span<T>)`, so not every
+   combination can work.
+
+We believe it's much easier if we enable code to use multi-targeting (that is,
+compile the same code for multiple platforms, like we do today.
+
+### Is .NET 5 a superset of .NET Framework 4.x?
+
+No, .NET 5 isn't a *superset* of .NET Framework. However, .NET 5 is
+the *successor* of both .NET Core 3.x as well as .NET Framework 4.x. Starting
+with .NET Core 3, you can build the same kind of workloads that you can with
+.NET Framework. Some of the tech has changed, but that's the way it's [going to
+be forever](https://devblogs.microsoft.com/dotnet/net-core-is-the-future-of-net/).
+The remaining delta will never be closed. Existing apps can stay on .NET
+Framework and will be supported, but we already said we'll no longer add new
+features to it.
+
+Thus, new apps should start on .NET Core. By branding it as .NET 5, this
+recommendation is much more obvious to both existing customers and new
+customers.
+
+### ~~Why are the OS specific flavors not versioned by the OS?~~
+
+*No longer applicable as we decided to allow that.*
+
+There are a couple of reasons why this isn't desirable:
+
+1. **It results a combinatorial explosion**. A TFM in the form of
+   `net5.0-windows7` would (syntactically) make any combination of .NET and OS
+   possible. This raises the question which combinations are supported, which
+   puts us back into having to provide the customer with a decoder ring.
+
+2. **It can make asset selection ill-defined**. Suppose the project is targeting
+   `net7.0-windows10`. The package offers `net5.0-windows10.0` and
+   `net6.0-windows7.0`. Now neither asset would be better.
+
+3. **A single version isn't enough**. Logically, you need at least two version
+   numbers to support OS targeting:
+
+      * *Minimum OS API Level*. Consumers of the library must use an SDK that
+        support this version, or a higher version. This allows library authors
+        to use types from the OS bindings in their public API surface without
+        causing type resolution issues for the consumer.
+
+      * *Minimum OS version*. For OS calls, it's common to guard OS calls with
+        version checks. This allows library authors to light up for later OS
+        versions without having to produce multiple binaries. In order for that
+        to be sound, a project generally cannot have lower version than whatever
+        its libraries support, unless the consumer guards their calls into that
+        library.
+  
+   Some platforms, such as Android, also have the notion of a *targeted OS
```

Comment:
Windows also have `MinVersion`, `MaxVersion` and `MaxVersionTested`.

Reply:
> Windows also have `MinVersion`, `MaxVersion` and `MaxVersionTested`.

Android also has a third version. @mhutch, should we standarize properties beyond `TargetPlatformMinimumVersion`?

Reply:
I think Android's Targeted OS version is equivalent to Windows' `MaxVersionTested`.